### PR TITLE
Try to configure cassandra for ssl

### PR DESCRIPTION
**Pull request by timyates**
Follow some instructions to generate truststore and keystore files. Grab the configuration for Cassandra 4.x and update the config to point to these files. Copy the files to the container

Still doesn't work

### CODE REVIEW

File: tests/cassandra-ssl/src/test/resources/ssl-cassandra.yaml [hunk 1]
```diff
@@ -0,0 +1,1821 @@
...
+  truststore_password: cassandra
+  # More advanced defaults:
+  protocol: TLS
+  store_type: JKS
+  cipher_suites: [
+     TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384, TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,
+     TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256, TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,
+     TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA, TLS_RSA_WITH_AES_128_GCM_SHA256, TLS_RSA_WITH_AES_128_CBC_SHA,
+     TLS_RSA_WITH_AES_256_CBC_SHA
+  ]
```
Comment (timyates) [inline tests/cassandra-ssl/src/test/resources/ssl-cassandra.yaml:1396]: This is configuring client SSL on the server

**Comment by timyates** [comment]
@wetted I think this works now...

I think the issue was that [`truststore` doesn't have a hyphen in it](https://github.com/micronaut-projects/micronaut-cassandra/pull/404/commits/661fc6278382efd0c901d15cbb9d364b8dc1c733#diff-6e78101e8d124891a5b4f581eae3ad17f902f7b4283e9ba0372542ea6a98a090L48) and I [think the class has to be a full path](https://github.com/micronaut-projects/micronaut-cassandra/pull/404/commits/661fc6278382efd0c901d15cbb9d364b8dc1c733#diff-6e78101e8d124891a5b4f581eae3ad17f902f7b4283e9ba0372542ea6a98a090R47)... not sure on this though

I also found this which helped https://ralph.blog.imixs.com/2022/10/22/setup-a-public-cassandra-cluster-with-docker/

**Review by wetted — APPROVED** [review]

**Comment by wetted** [comment]
Thank you for the contribution @timyates. You've been a big help.

**Comment by timyates** [comment]
@wetted Always a pleasure 👍

**Pull request merged**